ajax/libs/Flux-Slider/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "Flux-Slider",
+  "version": "1.4.4",
+  "filename": "js/flux.min.js",
+  "description": "CSS3 Animation based image transitions",
+  "homepage": "http://www.joelambert.co.uk/flux/",
+  "keywords": [
+    "css3",
+    "animation",
+    "image",
+    "transitions",
+    "plugin"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/joelambert/Flux-Slider"
+  },
+  "license": "MIT",
+  "author": "Joe Lambert <@joelambert> (http://www.joelambert.co.uk/)",
+  "autoupdate": {
+    "source": "git",
+    "target": "https://github.com/joelambert/Flux-Slider.git",
+    "fileMap": [
+      {
+        "basePath": "js",
```

Comment:
It's not good to use `js` or `css` as a basePath. Please modify it as the following:
```
"basePath": "",
"files": [
  "js/flux*",
  "css/style.css"
]
```